### PR TITLE
ghc-ppc-bootstrap: add openmaintainer, use https for homepage

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Non-functional changes: fix formatting of maintainer, add `openmaintainer`, use https with the homepage. Change tabs to spaces in multiples of four.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.6.8 Server
Xcode 3.2.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by mascguy** [comment]
Sergey, before submitting any new PRs, I'd like to ask that you complete all of the existing ones. (And wait until they've been merged.)

We simply cannot keep up with all of these. And frankly, it's starting to become exhausting for myself and others.

**Comment by mascguy** [comment]
> > Sergey, before submitting any new PRs, I'd like to ask that you complete all of the existing ones. (And wait until they've been merged.)
> 
> @mascguy Will be done. Sorry.

Not a big deal, just asking that you try not to have 50 open PRs at once!  ;-)

As for this one, you needn't close it, unless you've decided it's not needed...?

**Comment by barracuda156** [comment]
@mascguy Yeah, I got it. I won’t open anything until existing are cleared up. Unexpectedly short on time these days, and some of the stuff left opened is tricky to fix across the board.
I may need advice on a couple of existing PRs in regard of fixing for newer Intel systems (like `hugs98` and `factor`.

This PR fixes compliance with Macports standards for portfiles, but does not carry any functional improvements.

**Review by mascguy — APPROVED** [review]

**Pull request merged**
